config/fs.go

```diff
@@ -56,6 +56,7 @@ type FSConfig struct {
 	NoPrometheus                   bool   `toml:"no_prometheus"`
 	MountTimeoutSec                int64  `toml:"mount_timeout_sec"`
 	FuseMetricsEmitWaitDurationSec int64  `toml:"fuse_metrics_emit_wait_duration_sec"`
+	SkipCheckSnapshotterSupported  bool   `toml:"skip_check_snapshotter_supported"`
```

Comment:
FS sounds like file system

Reply:
what is FuseConfig then?

Reply:
IIUC FuseConfig is for the FUSE drivers while FSConfig is our implementation of a FUSE filesystem. You're right that it's a filesystem, but the options here are more for how the filesystem interacts with the snapshotter. IMO, since this option changes only the snapshotter startup and not the actual filesystem behavior, it should go there.

Reply:
ok fair enough will update it